website/docs/events/m4.md

```diff
@@ -0,0 +1,59 @@
+# Conference: Multiplier Event 4: Open Education
+
+Good education is always a challenge, even more so in the digital age, where information is readily available and the potential of automation is disrupting existing methods.
+Considering this, during the past years, at Open Education Hub, we worked on the following items:
+
+* we created on a methodology to develop and deploy open educational materials
+* we developed the infrastructure used for this and for automating items in the teaching process
+* we populated sample repositories, that we have used in the teaching process in university classes and in summer school
+
+We want to capture feedback on these items, to raise awareness and to encourage teachers and trainers to use them, as open source resources.
+
+So, on **Wednesday, September 13, 2023, 09:00-16:00 EEST**, we are organizing a multiplier event / conference to present our approach on developing and deploying digital educational materials as Open Education Resources (OER), using the open source collaborative model.
+As part of the event we are organizing practical workshops to get a hands-on experience of our work, and brainstorming meetings to get your input on important questions regarding the future of education in the digital age.
+Our goal is to extract best practices, guides and lessons learned in order to build up the methods, approaches and infrastructure for the future of education, a future that will be tied to digitalization and technology.
+
+The event takes place in the [PRECIS Center](https://goo.gl/maps/cBXziUixXDXNfxko6), Faculty of Automatic Control and Computers, University POLITEHNICA of Bucharest.
+
+We will start with a conference style set of presentations, continue with practical workshops, have lunch and then do brainstorming sessions.
+We will have coffee & drinks freely available.
+
+If you are an educator with interest in improving the quality of educational materials and the process of using them to teach, evaluate and support learning, this is an event you want to be part of.
+
+If you plan to attend, please fill [this form](https://forms.gle/nLmRaEcYTbP7zyiC6).
+
+## Schedule
+
+* Date: Thursday, June 22, 2023
+* Time: 15:00-17:30 EEST (Romania time)
+* Place: Room PR105, [PRECIS Center](https://goo.gl/maps/cBXziUixXDXNfxko6), Faculty of Automatic Control and Computers, University POLITEHNICA of Bucharest
```

Comment:
How is PR105 the place when the talks, brainstormings and workshops take place in PR001, PR002, PR003, PR103, and PR106? Why not simply meet in PR001?

Reply:
My mistake, I fixed it.